pkg/serviceaccount/credentials/credentials.go

```diff
@@ -68,7 +72,8 @@ func GetDefaultPath(outputFormat string) (filePath string) {
 // in the specified output format
 func Write(output string, filepath string, credentials *Credentials) error {
 	fileTemplate := getFileFormat(output)
-	fileBody := fmt.Sprintf(fileTemplate, credentials.ClientID, credentials.ClientSecret)
+	fileBody := fmt.Sprintf(fileTemplate, credentials.ClientID,
```

Comment:
We should totally start using the template package at some point, maybe a Hacktoberfest issue?

https://pkg.go.dev/text/template

Reply:
IMHO this is very clean and simple for now. Templates will just externalize/complicate integration